### DISCUSSION TITLE
"Error: This action with HTTP GET is not supported."

### DISCUSSION THREAD

**Discussion by xLEpEX** [nextauthjs/next-auth#9931] [Help]
hello,

im very new to nextauth, so i'm not sure if im missing something. I'have clone the [NEXT_AUTH_EXAMPLE_PROJECT](https://github.com/nextauthjs/next-auth-example) and added the TwitchProvider to the NextAuthConfig at the auth.ts file. To get a bether under understanding how to use nextauth.

On the first try with the nextauth 'latest' tag trying to login through the TwitchAuthProvider have received the error  'TypeError: next_dist_server_web_exports_next_request__WEBPACK_IMPORTED_MODULE_0__ is not a constructor' on the server logs which have been described [in this discussion](https://github.com/nextauthjs/next-auth/issues/9922).  After reading through the discussion i have updated the tag to beta to see if its working, and i'm now getting the error that i have written in the title "Error: This action with HTTP GET is not supported.". Can anyone tell me if i'm missing something?

**Comment by jeroengui** [comment]
Removing `NEXTAUTH_URL="http://localhost:3000"` from .env and .env.local fixed the issue for me.

**Reply by bereta80m** [reply]
thank you very much man, that works
